tf_encrypted/protocol/pond.py

```diff
@@ -989,6 +989,77 @@ def dispatch(self, base_name, *args, container=None, **kwargs):
         else:
             raise TypeError("Don't know how to {}: {}".format(base_name, [type(arg) for arg in args]))
 
+    @memoize
+    def zeros(
+        self,
+        shape,
+        tensor_type: str = "private",
+        apply_scaling: bool = True,
+        name: Optional[str] = None,
+        factory: Optional[AbstractFactory] = None
+    ) -> Union['PondPublicTensor', 'PondPrivateTensor', 'PondMaskedTensor']:
+
+        if tensor_type == "public":
+            out = _zeros_public(self, shape, apply_scaling, name, factory)
```

Comment:
Hey Yann, you can just check the tensor type here like: https://github.com/mortendahl/tf-encrypted/blob/master/tf_encrypted/protocol/pond.py#L869

Reply:
Hey Justin, how would you do that? zeros doesn't take a tensor as an input. It just takes a shape as an input then create a tensor with zeros based on this shape. So would you still add a tensor as an input like `def zeros(self, shape, input: PondTensor,` thanks for your review.

Reply:
Oh sorry didn't notice this didn't take any input. Jumped the gun!

Reply:
Wonder if there is another way to pass in a type instead of a string though.

Reply:
No worries, thanks for your question. good point, let me check

Reply:
Hey, wondering if we actually need an explicit masked version of this function or if public and private would be sufficient...feel like if we needed it masked for any reason it would be lifted at that point

Reply:
That's a good point. Let me remove mask.